canonical-web-and-design/jaas.ai
# Decommission store parts

## Done
Remove topics pages
Remove the store module from the site

## QA
- Open the demo
- Check homepage loads ok
- Go to /how-it-works
- Go to /jaas
- Check they all work
- Then check the following redirect to meaningful places on charmhub:
  - /openstack
  - /big-data
  - /containers
  - /kubernetes

## Details
Fixes https://github.com/canonical-web-and-design/app-tribe/issues/512
